File: server/dotnet/sdk-dotnet-example/Infrastructure/Mappers/PaymentDetails/CustomerBankAccountMapper.cs
using CustomerBankAccountSdk = OnlinePayments.Sdk.Domain.CustomerBankAccount;
using CustomerBankAccountDto = Business.Domain.Payments.PaymentDetails.CustomerBankAccount;

namespace Infrastructure.Mappers.PaymentDetails;

public static class CustomerBankAccountMapper
{
    public static CustomerBankAccountDto Map(CustomerBankAccountSdk? response)
    {
        return new()
        {
            AccountHolderName = response?.AccountHolderName,
            Bic = response?.Bic,
            Iban = response?.Iban
        };
    }
}

File: server/php/src/Infrastructure/Mappers/PaymentDetails/CustomerBankAccountMapper.php
<?php

namespace OnlinePayments\ExampleApp\Infrastructure\Mappers\PaymentDetails;

use OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails\CustomerBankAccount as CustomerBankAccountDto;
use OnlinePayments\Sdk\Domain\CustomerBankAccount as CustomerBankAccountSdk;

final class CustomerBankAccountMapper
{
    public static function mapFromSdkResponse(?CustomerBankAccountSdk $response): ?CustomerBankAccountDto
    {
        if ($response === null) {
            return null;
        }

        $dto = new CustomerBankAccountDto();
        $dto->accountHolderName = $response?->accountHolderName ?? null;
        $dto->bic = $response?->bic ?? null;
        $dto->iban = $response?->iban ?? null;

        return $dto;
    }
}
